GitUI/CommandsDialogs/FormPull.cs

```diff
@@ -97,6 +97,8 @@ public sealed partial class FormPull : GitModuleForm
         private readonly TranslationString _formTitleFetch = new TranslationString("Fetch ({0})");
         private readonly TranslationString _buttonPull = new TranslationString("&Pull");
         private readonly TranslationString _buttonFetch = new TranslationString("&Fetch");
+
+        private readonly TranslationString _pullFetchPruneAllConfirmation = new TranslationString("Warning! The fetch with prune will remove all the remote-tracking references which no longer exist on remotes. Do you want to proceed?");
```

Comment:
The display of the warning must be configurable
https://github.com/gitextensions/gitextensions/blob/f6929f477c8194dcf3ad3179df3572b9029bf18a/GitUI/CommandsDialogs/FormBrowse.cs#L2421-L2448

Reply:
Display of the warning is controlled by `AppSettings.DontConfirmFetchAndPruneAll`, which can be set in `Settings` > `Advanced` > `Confirmations`. I didn't change anything in this regard.

![fetchpruneall](https://user-images.githubusercontent.com/46861028/53687896-208d3280-3d3b-11e9-8ec3-2a5d05a30e15.png)

Thinking about it, maybe the text should be changed to something like " Fetch and prune branches", because now it applies also to a single repo and no longer necessarily to all repos at once?

Reply:
Confirm that it is working as intended, thank you

Reply:
> maybe the text should be changed to something like "Fetch and prune branches",

Sounds good, the suggestion is better than the existing text.